examples/dynamo/eg6/omp_script.py

```diff
@@ -22,5 +22,6 @@ def trans(psy):
 
     # take a look at what we've done
     schedule.view()
+    schedule.dag(file_format="png")
```

Comment:
I agree, I will modify the implementation so that nothing is output if the python graphviz bindings are not available. We should write out a warning to a log file in this case. I will add a comment in the code to state this.